composer.json

```diff
@@ -26,10 +26,13 @@
     "require": {
         "php": ">=5.3.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^5.5"
```

Comment:
For now, we should use PHPUnit 4.8, because Markdownify is compatible with PHP 5.3+
This is why Travis-ci is broken :radio: